pkg/controllers/status/cluster_status_controller.go

```diff
@@ -329,13 +428,13 @@ func getClusterAllocatable(nodeList *corev1.NodeList) (allocatable corev1.Resour
 	return allocatable
 }
 
-func getUsedResource(podList *corev1.PodList) corev1.ResourceList {
+func getUsedResource(podList []*corev1.Pod) corev1.ResourceList {
 	var requestCPU, requestMem int64
-	for podIndex, pod := range podList.Items {
+	for _, pod := range podList {
 		if pod.Status.Phase == "Running" {
 			for _, c := range pod.Status.Conditions {
 				if c.Type == "Ready" && c.Status == "True" {
-					podRes := addPodRequestResource(&podList.Items[podIndex])
+					podRes := addPodRequestResource(pod)
```

Comment:
It is different. The parameter of this function `getUsedResource ` has been changed to `podList []*corev1.Pod` because of unstructure conversion. So the loop range will only copy the pointer instead of the obj.

Reply:
Wow, you are right.